### PR TITLE
Sean / fix: broken create new demo account button for various languages on certain views

### PR DESCRIPTION
**Pull request by sean-binary**
Changes:

-   Fix: broken `create new demo account` button for various languages on certain views

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/layout/nav.js [hunk 1]
```diff
@@ -54,17 +54,17 @@ export const NavWrapper = styled.div`
     top: 0;
     z-index: 100;
 
-    .fresnel-between-start-tabletL {
+    /* .fresnel-between-start-bp1060 {
         display: none;
-    }
+    } */
```
Comment (suthesan-deriv) [inline src/components/layout/nav.js:59]: maybe we can remove these commented lines if not needed?

File: src/components/layout/nav.js [hunk 2]
```diff
@@ -54,17 +54,17 @@ export const NavWrapper = styled.div`
...
-    @media ${device.tabletL} {
-        .fresnel-between-start-tabletL {
+    /* @media ${device.bp1060} {
+        .fresnel-between-start-bp1060 {
             display: flex;
             width: 100%;
             align-items: center;
         }
-    }
+    } */
```
Comment (suthesan-deriv) [inline src/components/layout/nav.js:67]: same here?

File: src/components/layout/nav.js [hunk 3]
```diff
@@ -266,7 +259,7 @@ const LinkSignupButton = styled(LinkButton)`
 const HamburgerMenu = styled.img`
     cursor: pointer;
     display: none;
-    @media ${device.tabletL} {
+    @media (max-width: 1060px) {
```
Comment (yashim-deriv) [inline src/components/layout/nav.js:262]:
```suggestion
    @media ${device.bp1060} {
```

File: src/components/layout/nav.js [hunk 4]
```diff
@@ -275,7 +268,7 @@ const HamburgerMenu = styled.img`
 const CloseMenu = styled.img`
     cursor: pointer;
     display: none;
-    @media ${device.tabletL} {
+    @media (max-width: 1060px) {
```
Comment (yashim-deriv) [inline src/components/layout/nav.js:271]: ditto. should use the device constant that you declare for clarity

File: src/components/layout/nav.js (hunk 3 above)
Comment (sean-binary) [reply src/components/layout/nav.js:262]: This will need to be added into `device.js`, but sounds good 👍

**Review by yashim-deriv — DISMISSED** [review]

**Review by kevinw-binary — APPROVED** [review]

**Pull request merged**